packages/Operators/src/Factory/DTK_PointCloudOperatorFactory.hpp

```diff
@@ -90,7 +90,8 @@ class PointCloudOperatorFactory
     enum PointCloudOperatorType
     {
 	SPLINE_INTERPOLATION,
-	MOVING_LEAST_SQUARE
+	MOVING_LEAST_SQUARE,
+        NODE_TO_NODE
     };
```

Comment:
what's the point of the ``enum``s here? why don't you use directly the ``std::string`` as the key for the factory?